ospf6d/ospf6_main.c

```diff
@@ -172,6 +173,7 @@ static const struct frr_yang_module_info *const ospf6d_yang_modules[] = {
 	&frr_interface_info,
 	&frr_route_map_info,
 	&frr_vrf_info,
+	&frr_ospf_route_map_info,
```

Comment:
Let's rename to `frr_ospf6_route_map_info` to not confuse with ospfd structure.
(Also in `ospf6d/ospf6_routemap_nb.c/h`)

Reply:
Done